cmake/app/boilerplate.cmake

```diff
@@ -442,6 +442,11 @@ set(CMAKE_CXX_COMPILER_FORCED 1)
 
 include(${ZEPHYR_BASE}/cmake/host-tools.cmake)
 
+# Include board specific dtc flags and export them to python via system
+# variables to where device tree is parsed.
+include(${BOARD_DIR}/extra.cmake OPTIONAL)
+set(ENV{EXTRA_DTC_FLAGS} "${EXTRA_DTC_FLAGS}")
```

Comment:
I wonder if this mechanism might be a bit general for just changing a single flag. Feels like it might get abused.

@SebastianBoe 
Is there some other way to deal with this, if needed?

Reply:
It is overly general, but there is no better way. In practice, this has not proven to be a problem for board.cmake, and other hooks.

Reply:
@ulfalizer @SebastianBoe i didn't find a better way to pass the information to python. Will leave it as is since it's similar to board.cmake